### DISCUSSION TITLE
"Server controls" / "Restart" option should be easier to locate

### DISCUSSION THREAD

**Discussion by madbrain76** [home-assistant/frontend#12018] [Other feature requests]
The first few times, I restarted HAOS by pulling the power plug from my Pi.

I knew there had to be an option to restart HAOS. I simply couldn't figure out where it was in the UI.
It is currently located under Configuration / Settings / Server controls / Home Assistant / Restart .
Without the help of Google, I don't think I would ever have found it, short of exploring every single screen, sub-menu option, and tab.

I submit that "restarting home assistant" is an action. It's not part of the configuration, and it's not a setting.

It definitely falls under the category of "Server controls", but there is no way to guess that this is buried under Configuration / Settings .

I'm not a UX expert, and don't have any suggestion as to how it should be. But I think the current location is very much non-obvious, and this should be improved.

**Comment by frenck** [comment]
Some tips:

- There is a "search" in the top right of the configuration screen.
- Press `c` (for `c`ommand) anywhere in Home Assistant to trigger the QuickBar

**Comment by madbrain76** [comment]
Thank you very much for your response. I did see the magnifying glass in the configuration screen on the top right. This icon only appears on the configuration screen. Since I believed "restart" is an action rather than a configuration setting, I didn't think to click on this icon, which I thought would search the configuration.

I thought wrong. As it turns out, this search icon searches the entire UI, not just the configuration. But the magnifying glass is only shown on the configuration screen. This was confusing to me. Perhaps it should be on the left pane so it's always accessible, and it's obvious that it searches everything.

re: typing "c" . I definitely missed the tip about the tip, which sometimes shows up at the bottom of the "configuration" screen, and says "tip! press 'c' on any page to open this search bar."

When the tip appear, the search bar is not actually being shown yet, so "this search bar" is wrong. "the search bar" would be correct.

Also, "c" doesn't work universally. When I click any of the tabs on the left, including configuration, "c" has no effect. I have to click within the page on the right first, before "c" actually works.

Finally, why "c" to access the search function ? I'm guessing it stands for "configuration". But again, the search is for everything, both in the UI and configuration.

These are all separate UI issues. I will open separate bugs for them, and perhaps try to fix some.

**Reply by bschatzow** [reply]
I agree that there are many items difficult to find( especially the ones you use rarely). I believe the "C" is helpful but it still needs work. I find "C" not working sometimes.  Sometimes I have no idea what to search? As you found restart is not logical to you  I was searching where to add a device to energy. This took me a long time to find it under dashboards. What is logical to the developers is not always logical to the users.

**Reply by matthiasdebaat** [reply]
We're improving the Quick Bar by making it easier to find and turning it into a [single search bar](https://github.com/home-assistant/frontend/discussions/11511). In one of our concepts, we have the search option in the side menu. This isn't final yet, but I agree it's the most logical place. Making it into a single search bar we have to streamline the tip and keyboard shortcut for example cmd + K on Mac.

The bug that you need to focus on the page content to open the sidebar should indeed get a fix.

**Reply by madbrain76** [reply]
Matthias, thanks for your response ! The single search bar project looks like the right thing to do.
